torch/nativert/executor/memory/AliasAnalyzer.cpp

```diff
@@ -96,7 +113,7 @@ void AliasAnalyzer::maybe_update_aliases_from_schema(
         VLOG(1) << node.target()
                 << " may contain input/output alias: " << input->id() << " -> "
                 << output->id();
-        aliases_[output].insert(input);
+        aliases[output].emplace(input);
```

Comment:
I have asked you before, but I don't remember now, and I didn't find definition of  
c10::FastMap<const Value*, c10::FastSet<const Value*>> anywhere in this file. 

say, you have c = op(a, b), when are you going to have both a and b an alias of c?